### DISCUSSION TITLE
Builtin EMQX durable storage backend license

### DISCUSSION THREAD

**Discussion by AntonSmolkov** [emqx/emqx#13359] [General]
Hi!
EMQX 5.7.0 open source allows to enable and use builtin replicated durable storage backend (RocksDB). But, at the same time i see BSL.txt file here - https://github.com/emqx/emqx/blob/master/apps/emqx_ds_builtin_raft/BSL.txt
(Main branch, not 5.7.0 tag)

Does that mean that this functionality will become paid and only available in the enterprise version only in the future?

**Comment by wivwiv** [comment]
We plan to make some adjustments to durable storage in EMQX 5.8.0 Open source.  Not removing it, the open source version will still have durable storage. It helps improve the reliability of EMQX, which is something we value.
After adjustments, the Open Source Edition ensures the durability of the data on the local node. Based on this, we will invest more effort in the Enterprise Edition to implement replication and redundancy capabilities.

**Reply by wivwiv** [reply]
Of course, there will be other changes in the 5.8.0 open source, which we are preparing announcements for later.

**Reply by AntonSmolkov** [reply]
Thanks for the answers and for the really cool feature!

Doesn’t that mean that since 5.8.0 open source version will only have non-replicated durable storage like VerneMQ does?

**Reply by wivwiv** [reply]
Yes, it is.
